projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts

```diff
@@ -374,4 +382,15 @@ export class AnalyticsBoardComponent implements OnInit, AfterViewInit, OnChanges
         }
       });
   }
+
+  private getParamValue(param: string): string {
+    let paramValue = null;
+    const url = window.location.href;
+    const regex = new RegExp('[?&]' + param + '(=([^&#]*)|&|#|$)');
+    const results = regex.exec(url);
+    if (results && results[2]) {
+      paramValue = results[2];
+    }
+    return paramValue;
+  }
```

Comment:
It seems  that this method is never used ?

Reply:
it is used in line 175